common/math_test.go

```diff
@@ -0,0 +1,84 @@
+package common
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+func evaluateResult(t *testing.T, result []string, expected []string, message string) {
```

Comment:
These tests moved with `Complement`

Reply:
I had trouble running the tests locally - could we add some instructions to the README on how to run them?